pokemonred_puffer/environment.py

```diff
@@ -818,7 +822,7 @@ def get_explore_map(self):
         explore_map = np.zeros(GLOBAL_MAP_SHAPE)
         for (x, y, map_n), v in self.seen_coords.items():
             gy, gx = local_to_global(y, x, map_n)
-            if gy >= explore_map.shape[0] or gx >= explore_map.shape[1]:
+            if gy >= explore_map.shape[0] or gy < 0 or gx >= explore_map.shape[1] or gx < 0:
```

Comment:
```suggestion
            if 0 > gy >= explore_map.shape[0] or  0 > gx >= explore_map.shape[1]:
```

Yes you can do this in Python :)

Reply:
fixed!